### PR TITLE
one js question added

### PR DESCRIPTION
**Pull request by divyanshgupta003**
Added 6th question in js list

### CODE REVIEW

File: javascript/answers/q6.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What will be the output of the code below?">
```
Comment (arpansac) [inline javascript/answers/q6.html]: Please change the text of content attribute in meta tag

**Review by arpansac — CHANGES_REQUESTED** [review]

**Comment by divyanshgupta003** [comment]
I have done the required changes, Please view it again

**Comment by dakshkhetan** [comment]
Hi @divyanshgupta003, please take a fresh pull (from upstream master) and then make the changes.

**Comment by divyanshgupta003** [comment]
hey, made the changes now

**Comment by dakshkhetan** [comment]
@divyanshgupta003 you haven't taken a fresh pull from upstream. Please do that first!
Run command 'git pull upstream master'

**Comment by divyanshgupta003** [comment]
It's showing Already up to date

**Comment by dakshkhetan** [comment]
in the terminal run, git remote add upstream https://github.com/CodingNinjasCodes/InterQues.git
What does it say?

**Comment by divyanshgupta003** [comment]
fatal: remote upstream already exists.

**Comment by dakshkhetan** [comment]
Contact me on Slack.

**Comment by divyanshgupta003** [comment]
How to contact you on slack?

**Comment by shivamyth** [comment]
Update the PR otherwise will have to close it

**Comment by divyanshgupta003** [comment]
I have updated everything please check it again

File: javascript/answers/divyansh-gupta-q7.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                </h3>
+                
+            </div>
+            <div>
+                <p>
+                    ‘Strict mode’ is a restricted variant of JavaScript. <br>
+                    But in ‘Strict mode’ it will throw all types of errors, even the silent errors. 
+                    Thus, the process of debugging becomes easier. And the chances for making mistake for the developer is reduced.
+                </p>
+                <strong>Answer: Given below are the characteristics of JavaScript ‘Strict Mode’:</strong>
```
Comment (shivamyth) [inline javascript/answers/divyansh-gupta-q7.html:52]: Write a js program with and without strict mode to make it understand better

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/divyansh-gupta-q7.html (hunk 2 above)
Comment (divyanshgupta003) [reply javascript/answers/divyansh-gupta-q7.html:52]: Yeah I have changed the definition and also added a simple example for it

**Review by arpansac — DISMISSED** [review]
I deleted one extra file

**Review by arpansac — APPROVED** [review]

**Pull request merged**